src/js/letters/containers/AddressSection.jsx

```diff
@@ -295,9 +301,16 @@ export class AddressSection extends React.Component {
           <div className="letters-address street">{streetAddress}</div>
           <div className="letters-address city-state">{cityStatePostal}</div>
           <div className="letters-address country">{country}</div>
+          <button className="address-help-btn" onClick={this.openAddressHelp}>What is This?</button>
```

Comment:
Is there a class that we can reuse to style this like a link?

Reply:
Mm, kind of - it's `usa-button-unstyled` but its padding / margins, outline and coloring are off, so would have needed to modify it pretty heavily anyway.

Reply:
Gah, good catch. The issue description does use a lowercase 't'.